src/mongo/db/pipeline/document_source_group.cpp

```diff
@@ -66,7 +66,7 @@ DocumentSource::GetNextResult DocumentSourceGroup::getNext() {
         invariant(initializationResult.isEOF());
     }
 
-    for (auto&& accum : _currentAccumulators) {
+    for (auto && accum : _currentAccumulators) {
```

Comment:
It does look like you've somehow got the wrong formatting. Let's chat about what you did to format the code.